README.md

```diff
@@ -33,6 +33,7 @@ A list of all the resources one might find useful while developing Islamic softw
 * [Muslim Salat](http://muslimsalat.com/api/)
 * [PrayTimes](http://praytimes.org/code/)
 * [Al Adhan](https://aladhan.com/prayer-times-api)
+* [Muwaqqit app, API & docs](https://www.muwaqqit.com/index)
```

Comment:
Since this list is supposed to be for developer resources specifically, it should link to the API docs. My suggested change is:
```
* [Muwaqqit](https://docs.muwaqqit.com/#/api-v1)
```